toolium/utils/dataset.py

```diff
@@ -181,12 +181,10 @@ def _replace_param_replacement(param, language):
         replacements[date_expr] = _replace_param_date(date_expr, language)[0]
         
     new_param = param
-    param_replaced = False
     for key in replacements.keys():
         if key in new_param:
             new_param = new_param.replace(key, replacements[key])
-            param_replaced = True
-    return new_param, param_replaced
+    return new_param
```

Comment:
If we remove `param_replaced` from here, we should remove it also from `_replace_param_date`, where it is no longer used either. The other option is to maintain the current (new_value, has_changed) return scheme in all the `_replace_xxx` functions, just in case.